src/colejorz/__init__.py

```diff
@@ -1,19 +1,21 @@
 """Clearcode Colejorz."""
+from typing import Dict, Any
 from wsgiref.simple_server import make_server
 
 from pyramid.request import Request
 from pyramid.config import Configurator
 
+from colejorz.pilothouse import Pilothouse
 from colejorz.stationmaster import StationMaster
 from colejorz.views import get_state
 
 
-def get_stationmaster(request):
+def get_stationmaster(request: Request) -> Pilothouse:
```

Comment:
Are station masters and pilot houses the same thing? (Having two imports for two modules with such things suggests no.)